tests/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Configuration.Tests/ConfigUpdaterSubStepTests.cs

```diff
@@ -0,0 +1,246 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Autofac;
+using Autofac.Extras.Moq;
+using Microsoft.Extensions.Logging.Abstractions;
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Configuration.Tests
+{
+    public class ConfigUpdaterSubStepTests
+    {
+        [Fact]
+        public void CtorTests()
+        {
+            // Arrange
+            (_, var step) = GetMockContextAndStep(new[] { "a" }, true, new[] { new TestConfigUpdater(BuildBreakRisk.Medium, false) });
+
+            // Act
+            var risk = step.Risk;
```

Comment:
Do we need all the local variables?

Reply:
No, my 'Act' section was just feeling a little lonely :) I think you're right that it's better without them. I'll inline them.